plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestingIcebergPlugin.java

```diff
@@ -27,6 +27,7 @@
 public class TestingIcebergPlugin
         extends IcebergPlugin
 {
+    public static final boolean HIDE_DELTA_LAKE_TABLES = false;
```

Comment:
maybe "HIDE_DELTA_LAKE_TABLES_IN_ICEBERG"
so that staticly imported remains unambiguous 

(Hive, Iceberg and Delta connectors have overlapping classpaths, so we need to be extra careful)